README.md

```diff
@@ -80,32 +80,61 @@ metric exporters, service monitors, and custom dashboards.
 
-`USER_DIR` must refer to a directory. The directory can include any or all of
-the following files in the structure below. Additional content is allowed, but
-will be ignored.
+The `USER_DIR` environment variable enables you to use a directory outside of the local repository to contain the customization files. By using a location outside of the repository, you can preserve your customizations when you deploy new versions of the monitoring and logging applications. You could also put the customization files in a location that is under version control.
+
+For example, to create a customization directory named *my_custom_files* that contains directories for monitoring and logging customization files:
+
+```bash
+mkdir -p ~/my_custom_files/monitoring
+mkdir -p ~/my_custom_files/logging
+```
+
+### Specifying Customization Variables and Parameters
+
+After you create the location for your customization files, you can customize the components that you deploy for monitoring and logging by specifying environment variables and Helm chart parameters in this set of customization files (in this example, contained in the *my_custom_files* root directory):
 
 ```text
-- user.env
-- [monitoring]
-  - user.env
-  - user-values-prom-operator.yaml
-  - user-values-pushgateway.yaml
-- [logging]
-  - user.env
-  - user-values-elasticsearch-open.yaml
-  - user-values-es-exporter.yaml
-  - user-values-fluent-bit-open.yaml
+/my_custom_files/user.env
+
+/my_custom_files/monitoring/user.env
+/my_custom_files/monitoring/user-values-prom-operator.yaml
+/my_custom_files/monitoring/user-values-pushgateway.yaml
+
+/my_custom_files/logging/user.env
+/my_custom_files/logging/user-values-elasticsearch-open.yaml
+/my_custom_files/logging/user-values-es-exporter.yaml
+/my_custom_files/logging/user-values-fluent-bit-open.yaml
+```
+
+You specify the environment variables in the `user.env` files and the Helm chart parameters in the `*.yaml` configuration files. Note that you can also specify environment variables on a command line, but specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments. 
+
+You do not have to specify values in all of these files. The deployment process ignores any files other than these.
+
+This repository contains several sample configuration files which provide values for specific situations. You can copy the configuration files for a sample from the repository to your customization file directory  in order to provide you with a set of customizations that are valid for the situation. You can then make further modifications to the files. 
+
+If more than one sample applies to your environment, you can manually copy the values from the other sample files to the files in your customization directory.
+
+The [monitoring README](monitoring/README.md) and [logging README](logging/README.md) for information about determining valid values for the `user.env` and `*.yaml` files.
```

Comment:
Prefix this sentence with "See" or change to "...contain information about ..."

Reply:
Done